starboard/android/shared/player_components_factory.h

```diff
@@ -318,8 +321,8 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
                    << ". Tunnel mode is disabled.";
     }
 
-    if (kForceTunnelMode && !enable_tunnel_mode) {
-      SB_LOG(INFO) << "`kForceTunnelMode` is set to true, force enabling tunnel"
+    if (IsForceTunnelModeEnabled() && !enable_tunnel_mode) {
+      SB_LOG(INFO) << "`force_tunnel_mode` is set to true, force enabling tunnel"
```

Comment:
Format checking: change to
```
      SB_LOG(INFO)
          << "`force_tunnel_mode` is set to true, force enabling tunnel"
          << " mode.";
```

Reply:
Sorry for the incorrect operation that I accidentally closed this PR when I rebase the commit.
I will make a new PR with the correction